datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py

```diff
@@ -65,6 +66,11 @@ class OpenMetricsBaseCheck(OpenMetricsScraperMixin, AgentCheck):
         'request_size': {'name': 'request_size', 'default': 10},
     }
 
+    # Allow tracing for openmetrics integrations
+    def __init_subclass__(cls, **kwargs):
```

Comment:
Will this still work for python 2?

Reply:
Yes, this was added in python 3.6 and backported to python2 (can't find the changelog) ; also tests for this prs are running on python 2